docs/GettingStarted/configuration/PelorusCore.md

```diff
@@ -0,0 +1,235 @@
+
+### Prometheus credentials
+
+###### openshift_prometheus_htpasswd_auth
+
+- **Required:** no
+    - **Default Value:** internal:{SHA}+pvrmeQCmtWmYVOZ57uuITVghrM=
+- **Type:** string
+
+: Credentials for the `internal` user that are used by Grafana to communicate with the Prometheus instance deployed by Pelorus. Those credentials must use `internal` user name and must match the [openshift_prometheus_basic_auth_pass](#openshift_prometheus_basic_auth_pass) password from the [Grafana credentials](#grafana-credentials) configuration option. 
+
+: Format supported: Base64-encoded SHA-1
+
+: **Note:** The generate new password for the `internal` user, you may invoke the `htpasswd` CLI as in the example:
+```shell
+$ htpasswd -nbs internal <my-secret-password>
+```
+
+### Multiple Prometheus
+
+By default Pelorus gathers the data from the Prometheus instance deployed in the same cluster in whcih it is running. To collect data across multiple OpenShift clusters additional Prometheus hosts have to be configured. To do this `extra_prometheus_hosts` configuration option is used.
+
+###### extra_prometheus_hosts
+
+- **Required:** no
+    - **Default Value:** no
+- **Type:** list
+
+It is a list that consists of three configuration items per additional Prometheus host:
+: * id - a description of the prometheus host (this will be used as a label to select metrics in the federated instance).
+: * hostname - the fully qualified domain name or ip address of the host with the extra Prometheus instance
+: * password - the password used for the `internal` basic auth account (this is provided by the k8s metrics prometheus instances in a secret).
+: Example:
+```yaml
+extra_prometheus_hosts:
+- id: "ci-1"
+  hostname: "prometheus-k8s-openshift-monitoring.apps.ci-1.example.com"
+  password: "<redacted>"
+
+- id: "ci-2"
+  hostname: "prometheus-k8s-openshift-monitoring.apps.ci-2.example.com"
+  password: "<redacted>"
+```
+
+## Grafana
+
+Grafana is a dashboard which represents data stored in the [Prometheus](#prometheus). Is deployed as an Grafana Operator available from the OLM dependency mechanism.
+
+### Grafana credentials
+
+###### openshift_prometheus_basic_auth_pass
+
+- **Required:** no
+    - **Default Value:** changeme
+- **Type:** string
+
+: The password that grafana will use for its Prometheus datasource. Must match the [openshift_prometheus_htpasswd_auth](#openshift_prometheus_htpasswd_auth).
+
+## Thanos
+
+The Pelorus chart supports deploying a [Thanos](https://thanos.io/) instance for long term storage.
+
+###### thanos_version
+
+- **Required:** no
+    - **Default Value:** v0.28.0
+- **Type:** string
+
+: Which Thanos version from the [Official Thanos docker image](https://quay.io/repository/thanos/thanos) use.
+
+###### bucket_access_point
+
+- **Required:** no
+    - **Default Value:**
```

Comment:
I would remove the ones without a default value (just have the required and type items)